### DISCUSSION TITLE
How to override the children based on parent properly ?

### DISCUSSION THREAD

**Discussion by khanakia** [tailwindlabs/tailwindcss#9416] [Help]
I have a layout which i want to override based on parent selector e.g. list or grid.

So if parent has `list` selector class then change the child layout to list otherwise grid.

It's the same feature i want as tailwind uses for the dark mode but i cannot do like this i tried that

```
## Should be Green BG
<div class="list">
      <div class="bg-gray-100 list:bg-green-100">Test</div>
</div>

## Should be Gray BG
<div>
      <div class="bg-gray-100 list:bg-green-100">Test</div>
</div>
```

**Comment by wongjn** [comment]
You could look at using [arbitrary variants](https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants). In your example, this would be `[.list>&]:bg-green-100`.
